docs/intro/tutorial.rst

```diff
@@ -124,9 +124,9 @@ and defines some attributes and methods:
   is an instance of :class:`~scrapy.http.TextResponse` that holds
   the page content and has further helpful methods to handle it.
 
-  The :meth:`~scrapy.spiders.Spider.parse` method usually parses the response, extracting
-  the scraped data as dicts and also finding new URLs to
-  follow and creating new requests (:class:`~scrapy.http.Request`) from them.
+  The :meth:`~scrapy.spiders.Spider.parse` method usually parses the response: it extracts
+  the scraped data as dicts, finds new URLs to
+  follow and creates new requests (:class:`~scrapy.http.Request`) from them.
```

Comment:
I see as an improvement in general, although now it reads as a 3-item list instead of a 2-item list as before, so I would suggest adding an extra ‘and’ after the comma.

```suggestion
  The :meth:`~scrapy.spiders.Spider.parse` method usually parses the response: it extracts
  the scraped data as dicts, and finds new URLs to
  follow and creates new requests (:class:`~scrapy.http.Request`) from them.
```